worlds/sc2/items.py

```diff
@@ -1380,13 +1380,24 @@ def get_full_item_list():
     item_names.CORRUPTOR_RESOURCE_EFFICIENCY:
         ItemData(289 + SC2HOTS_ITEM_ID_OFFSET, ZergItemType.Mutation_3, 29, SC2Race.ZERG, parent_item=item_names.CORRUPTOR,
                  origin={"ext"}),
-
     item_names.PRIMAL_IGNITER_CONCENTRATED_FIRE:
         ItemData(290 + SC2HOTS_ITEM_ID_OFFSET, ZergItemType.Mutation_4, 0, SC2Race.ZERG, parent_item=item_names.ROACH_PRIMAL_IGNITER_ASPECT,
                  origin={"ext"}),
     item_names.PRIMAL_IGNITER_PRIMAL_TENACITY:
         ItemData(291 + SC2HOTS_ITEM_ID_OFFSET, ZergItemType.Mutation_4, 1, SC2Race.ZERG, parent_item=item_names.ROACH_PRIMAL_IGNITER_ASPECT,
                  origin={"ext"}),
+    item_names.TYRANNOZOR_TYRANTS_PROTECTION:
+        ItemData(292 + SC2HOTS_ITEM_ID_OFFSET, ZergItemType.Mutation_4, 8, SC2Race.ZERG, parent_item=item_names.ULTRALISK_TYRANNOZOR_ASPECT,
+                 origin={"ext"}),
+    item_names.TYRANNOZOR_BARRAGE_OF_SPIKES:
+        ItemData(293 + SC2HOTS_ITEM_ID_OFFSET, ZergItemType.Mutation_4, 9, SC2Race.ZERG, parent_item=item_names.ULTRALISK_TYRANNOZOR_ASPECT,
+                 origin={"ext"}),
+    item_names.TYRANNOZOR_IMPALING_STRIKE:
+        ItemData(294 + SC2HOTS_ITEM_ID_OFFSET, ZergItemType.Mutation_4, 10, SC2Race.ZERG, parent_item=item_names.ULTRALISK_TYRANNOZOR_ASPECT,
+                 origin={"ext"}),
+    item_names.TYRANNOZOR_HEALING_ADAPTION:
+        ItemData(295 + SC2HOTS_ITEM_ID_OFFSET, ZergItemType.Mutation_4, 11, SC2Race.ZERG, parent_item=item_names.ULTRALISK_TYRANNOZOR_ASPECT,
```

Comment:
Note if Envy is claiming the 2~7 slots in this category, that also likely means he's claiming the 292~297 IDs and you could end up with an ID conflict. It's an easier fix, but keep it in mind for the potential merge conflict (or let Envy know if you merge first)

Reply:
How does this work with the ids reaching 300?  It seems like the next block starts at 300.  would they conflict?

Reply:
They'd conflict, yeah. The only real important thing is that all the IDs are unique, though, we don't care exactly what they are.

Until very recently (0.4.6), Archipelago required all IDs to be unique across all games. Now with 0.5.0 we're free to put IDs anywhere and we just have to avoid colliding with ourselves. We want to maintain the IDs of existing items for backwards compatibility reasons, but new IDs can go basically everywhere. It may also be reasonable to introduce new base offsets for some categories that really grow a lot.

Reply:
I don't mind doing this, but I feel like someone more important than me might want to make a decision on how to handle it, since my way will probably be ugly and hated. (It would be to add 3 more constants each 1000 then the current ones.)

Reply:
we have a unit test to catch this